service/openapi.yaml

```diff
@@ -13,46 +13,29 @@ info:
 security:
   - ApiKeyAuth: [ ]
   - BearerAuth: [ ]
+tags:
+  - name: OFREP core
+    description: |
+      Core APIs to implement to support OFREP.  
+      **Required**  
+      
+      *This is the minimum set of APIs required for a flag management system to be OFREP compatible.*
+      
+  - name: OFREP extensions
+    description: |
+      Extension APIs to provide full support for OFREP.  
+      **Optional**  
+
+      *These APIs are utilized by the providers to enhance the OFREP experience.*
+      
```

Comment:
```suggestion
tags:
  - name: OFREP Core
    description: |
      **Required**: Core APIs to implement to support OFREP.  
      *This is the minimum set of APIs required for a flag management system to be OFREP compatible.*
  - name: OFREP Extensions
    description: |
      **Optional**: Extension APIs to provide full support for OFREP.  
      *These APIs are utilized by the providers to enhance the OFREP experience.*
      
```

I know naming is hard, but I think we could use OFREP Core & OFREP Extensions capitalized terms. And the formatting looks good ,

![image](https://github.com/open-feature/protocol/assets/8186721/51e0ac45-fe40-4b60-b814-244d468e0f6b)